### DISCUSSION TITLE
The label FourVector was not found when I used FeynCalc.

### DISCUSSION THREAD

**Discussion by AngelsAiben** [FeynCalc/feyncalc#207] [Q&A]
I built a model file with FeynRule. I hope to give the amplitude of a process and simplify it through FeynArts and FeynCalc. I put the model file into the Model folder of FeynArts. When I used to generate FeynArts amplitude and Feynman diagram in Mathematica, everything was normal, but when I wanted to simplify through FeynCalc, an error occurred: the label FourVector were not found. I don't know how to solve this problem.

**Comment by vsht** [comment]
I suppose that you forgot to patch your model, see here 

https://feyncalc.github.io/FeynCalcBookDev/Extra/CustomModels.html
